ts/features/bonus/bonusVacanze/store/reducers/__tests__/allActive.test.ts

```diff
@@ -10,9 +10,13 @@ import { BonusActivationStatusEnum } from "../../../../../../../definitions/bonu
 import { BonusCode } from "../../../../../../../definitions/bonus_vacanze/BonusCode";
 import { mockedBonus } from "../../../__mock__/mockData";
 import { ownedActiveOrRedeemedBonus } from "../allActive";
+import { ServicesPreferencesModeEnum } from "../../../../../../../definitions/backend/ServicesPreferencesMode";
 
 const fiscalCode = "ABCDEF83A12L719R" as FiscalCode;
 const profile: InitializedProfile = {
```

Comment:
I find 16 matches in this PR alone for `profile: InitializedProfile`. It may be worth to define it in a mock file and import it where is needed for testing to reduce the noise and the effort in future updates.
Nothing urgent nor a blocker for this PR.